src/Nevermind/Nevermind.Core/Crypto/PrivateKey.cs

```diff
@@ -39,6 +43,8 @@ private PublicKey ComputePublicKey()
 
         public Address Address => PublicKey.Address;
 
+        public Guid Id { get; set; }
```

Comment:
we will need to review how we pass private key around in memory (need to be protected and overwritten), separately - let us discuss why we need Guid - can't we store by public key? (or is it to cover all the scenarios when public key is same? - rare)